### PR TITLE
WiP: NTO: HyperShift tests

### PR DESCRIPTION
**Pull request by jmencak**
First attempt at enabling HyperShift e2e testing for NTO in OCP 4.12

### CODE REVIEW

**Comment by jmencak** [comment]
Hmm, `ci/prow/pr-rehearse`
```
time="2022-08-29T14:29:29Z" level=warning msg="hidden jobs are not allowed to be rehearsed" job=pull-ci-openshift-priv-cluster-node-tuning-operator-release-4.12-e2e-aws-operator-hypershift org=openshift repo=openshift-priv/cluster-node-tuning-operator
```
Not sure how I can test this here.  Any ideas @kpouget ?  I'd like to basically just debug the CI job online and take a look at the installed HyperShift cluster.  If this is not possible, I guess we could merge and test directly by creating a dummy NTO PR.

File: ci-operator/jobs/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-release-4.12-presubmits.yaml [hunk 1]
```diff
@@ -144,6 +144,60 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-aws-operator,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (kpouget) [inline ci-operator/jobs/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-release-4.12-presubmits.yaml:148]:
```
     always_run: true
```
dont' forget to double check if you want this enabled

**Comment by jmencak** [comment]
At the moment there is no good framework to do this via `openshift/release`, will re-visit this when there is.  Rely on the HyperShift project's e2e tests for the time being.

/close

**Pull request closed without merging**